src/vulkan-renderer/application.cpp

```diff
@@ -181,6 +181,9 @@ VkResult Application::load_octree_geometry() {
         child->indent(1, false, 2);
     }
 
+    cube->childs()[3]->rotate({0, 1, 0});
+    cube->childs()[6]->rotate({0, 0, 1});
```

Comment:
As I mentioned earlier, we could call this just as `cube->childs()[6]->rotate(ROTATION_AXIS_X);`.

Reply:
We will resolve this in another pull request: https://github.com/inexorgame/vulkan-renderer/issues/251